README.md

```diff
@@ -110,13 +100,30 @@ If you're using `docker-mailserver` version `v10.1.x` or below, you will need to
 
 #### First Things First
 
-**Use `docker-compose up / down`, not `docker-compose start / stop`**. Otherwise, the container is not properly destroyed and you may experience problems during startup because of inconsistent state.
-
-You are able to get a full overview of how the configuration works by either running:
```

Comment:
Why was the line break removed? The two sentences aren't exactly related and the 2nd sentence has an association to the ordered list that follows.

Reply:
I will have a look and revert it :)